decidim-comments/spec/models/comment_spec.rb

```diff
@@ -181,6 +181,15 @@ module Comments
           end
         end
 
+        describe "when the body contains HTML" do
+          let(:body) { %(<a target="alert(1)" href="http://subdomain1.portswigger-labs.net/xss/xss.php?context=js_string_single&x=%27;eval(name)//">XSS via target in a tag</a>) }
```

Comment:
We shouldn't depend on an external domain for running our specs, can you change it 🙏🏽 ?